src/components/VContentSwitcher/VContentItem.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <VItem
+    :selected="selected"
+    :is-first="itemId === 0"
+    @click.native="$emit('click', item)"
+  >
+    <div class="flex flex-row items-center py-3">
```

Comment:
```suggestion
    <div class="flex flex-row items-center py-3">
```

The `py-3` seems like it is making each item too tall, which is the main problem i notice when testing.